lib/build.js

```diff
@@ -412,14 +418,25 @@ const buildAct = config => ({act, oog, pass, name}) => {
   if (pass) {
       gasCond = "VGas " + (oog ? "<Int " : ">=Int ") + act.gas
   }
+
+  //The multiplication of the list monad  
+  const mu = ListOfList => ListOfList.reduce((a, list) => a.concat(list))
```

Comment:
this is usually called "flatten"

Reply:
also don't giving reduce an initial value (e.g. `[]`) will result in a crash whenever ListOfList is empty (e.g. `[]`)